flocker/node/script.py

```diff
@@ -154,13 +133,8 @@ class ZFSAgentScript(object):
     a Flocker cluster.
     """
     def main(self, reactor, options, volume_service):
-        try:
-            agent_config = options[u'agent-config']
-            configuration = yaml.safe_load(agent_config.getContent())
-        except IOError:
-            raise ConfigurationError(
-                "Configuration file does not exist at '{}'.".format(
-                    agent_config.path))
+        agent_config = options[u'agent-config']
```

Comment:
This was suggested on https://github.com/ClusterHQ/flocker/pull/1403 after that PR was merged.
It was just as quick to change it here as make a follow-up issue.